src/models/user.ts

```diff
@@ -1,5 +1,5 @@
 import { DataTypes } from 'sequelize';
-import UsersAttributes from '../types/index';
+import { UsersAttributes } from '../types/index';
```

Comment:
same note here, no need for type index
```suggestion
import { UsersAttributes } from '../types;
```